app/src/main/java/com/teamwss/websoso/ui/myActivityDetail/MyActivityDetailActivity.kt

```diff
@@ -0,0 +1,43 @@
+package com.teamwss.websoso.ui.myActivityDetail
+
+import android.os.Bundle
+import androidx.activity.viewModels
+import com.teamwss.websoso.R
+import com.teamwss.websoso.common.ui.base.BaseActivity
+import com.teamwss.websoso.databinding.ActivityMyActivityDetailBinding
+import com.teamwss.websoso.ui.myActivityDetail.adapter.MyActivityDetailAdapter
+import dagger.hilt.android.AndroidEntryPoint
+
+@AndroidEntryPoint
+class MyActivityDetailActivity :
+    BaseActivity<ActivityMyActivityDetailBinding>(R.layout.activity_my_activity_detail) {
+    private val myActivityDetailViewModel: MyActivityDetailViewModel by viewModels()
+    private val myActivityDetailAdapter: MyActivityDetailAdapter by lazy {
+        MyActivityDetailAdapter()
+    }
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setupMyActivitiesDetailAdapter()
+        setUpObserve()
+        setupBackToMyActivityButton()
+    }
+
+    private fun setupMyActivitiesDetailAdapter() {
+        binding.rvMyActivityDetail.apply {
+            adapter = myActivityDetailAdapter
+        }
+    }
+
+    private fun setUpObserve() {
+        myActivityDetailViewModel.myActivity.observe(this) { activities ->
+            myActivityDetailAdapter.submitList(activities)
+        }
+    }
+
+    private fun setupBackToMyActivityButton() {
```

Comment:
c: 
```suggestion
    private fun onBackButtonClick() {
```
뭐 요런 식은 어떨지! 리스너에 대한 컨벤션에 맞춰서 ?

Reply:
좋은 것 같아요! 수정하겠습니다!